features/admin.policy-administration.v1/components/policy-administration-page-layout.tsx

```diff
@@ -73,20 +72,24 @@ const PolicyAdministrationPageLayout: FunctionComponent<PolicyAdministrationPage
     const [ pageActive, setPageActive ] = useState<number>(0);
     const [ hasMoreActivePolicies, setHasMoreActivePolicies ] = useState<boolean>(true);
     const [ activePolicies, setActivePolicies ] = useState<PolicyInterface[]>([]);
+    const [ searchQuery, setSearchQuery ] = useState<string>("");
+    const [ submittedSearchQuery, setSubmittedSearchQuery ] = useState<string>("");
 
     const {
         data: inactivePolicyArray,
         isLoading: isLoadingInactivePolicies,
         error: inactivePolicyError,
         mutate: mutateInactivePolicy
-    } = useGetPolicies(true, pageInactive, false, "*", "ALL");
+    } = useGetPolicies(true, pageInactive, false, submittedSearchQuery &&
```

Comment:
Usually we use names like `usePolicies` for SWR hooks. Refer to other hooks as well.
```suggestion
    } = usePolicies(true, pageInactive, false, submittedSearchQuery &&
```

Reply:
useGetPolicies is the convention we use for SWR hooks. Therefore no need to change it.

Ref:https://github.com/wso2/identity-apps/tree/master/features/admin.actions.v1/api/use-get-action-by-id.ts